src/styles/main.scss

```diff
@@ -1,14 +1,6 @@
-html {
-  font-family: Roboto, sans-serif;
-}
-
-body {
-  margin: 0;
-}
-
-* {
-  box-sizing: border-box;
-}
 
+@import "./reset.scss";
+@import "./normalize.scss";
+@import "./utils.scss/mixins.scss";
 @import "_header.scss";
 @import "_catalog.scss";
```

Comment:
Should delete the underline in the file name,  underline better to use for util files